layouts/default.vue

```diff
@@ -67,6 +67,6 @@
 
 <style lang="scss" scoped>
   .theme--light.v-application {
-    background: var(--v-secondary-base) url('/background.jpg') no-repeat center top / cover;
+    background: #7E643A url('/blockchange/background.jpg') no-repeat center top / cover;
```

Comment:
Why are we hardcoding a color instead of using a variable?

Reply:
Because we didn't have a variable for this. Introduced it now.